src/playwrightTest.ts

```diff
@@ -97,14 +97,21 @@ export class PlaywrightTest {
   async listFiles(config: TestConfig): Promise<ConfigListFilesReport | null> {
     const configFolder = path.dirname(config.configFile);
     const configFile = path.basename(config.configFile);
-    const allArgs = [config.cli, 'list-files', '-c', configFile];
+    const allArgs = [config.cli, 'list-files', '-c', config.configFile];
     {
       // For tests.
       this._log(`${escapeRegex(path.relative(config.workspaceFolder, configFolder))}> playwright list-files -c ${configFile}`);
     }
-    const output = await this._runNode(allArgs, configFolder);
+
+    // Execute the list-files command with workspace folder as working directory and
+    // use full path to the config file to allow process.cwd() to work as expected
+    // in config files
+    const output = await this._runNode(allArgs, config.workspaceFolder);
     try {
-      return JSON.parse(output) as ConfigListFilesReport;
+      // Strip the JSON blob starting with {"projects":} from the output of the
+      // executed process
+      const strippedOutput = /{"projects":(.+)\}/ig.exec(output)?.[0] ?? ''
```

Comment:
Really hacky but it works in my circumstance. I am wondering if there is a better way to exchange the list of projects

Reply:
What exactly are we fixing here? Let's get issues filed for those two problems?